bundles/org.openhab.core.model.script/.classpath

```diff
@@ -18,11 +18,6 @@
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry including="**" kind="src" output="target/classes" path="resources">
```

Comment:
Hm, as this is a "model" bundle, we have "src" and "resource" as source folders and not "src/main/java" and "src/main/resources". I hence think that this PR is not correct.

Reply:
Just as a side note - this is the last OHC bundle where the `resources/` folder is not located in`src/main/`.

Does it matter the classpath entry uses `including="**"` whereas all other bundles use `excluding="**"` in this place?

Reply:
I stick to this PR. In any other case I got this message:

```
[INFO] --- maven-resources-plugin:3.0.2:resources (default-resources) @ org.openhab.core.model.script ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory ~/openhab-master/git/openhab-core/bundles/org.openhab.core.model.script/src/main/resources
```

and the config files are not copied into `target/classes/`

Reply:
Shouldn't it be as easy as https://github.com/openhab/openhab-core/pull/1160?
This makes Maven consider the "resources" folder and put its content into the jar.
Yes, the "skip non existing..." message will be shown, but this is only an INFO as Maven always checks for its default resource folder to be present and it wants to make sure that the user hasn't accidentially forgotten it. We can just ignore this message.